workflow/calculation/site_response_BB/site_template.yaml

```diff
@@ -0,0 +1,40 @@
+H_soil: 0.0  # Depth to bottom of soil column
+dampS_base: 0.0  # Vs damping ratio of layer to which you deconvolve
+
+numLayers: 0  # number of soil layers (not counting layer 0 which is bedrock)
+waterTable: 0.0  # if water not present set waterTable anywhere below depth of model
```

Comment:
Probably not a big thing but there is two types of styling here, camel case and some form of snake case

Reply:
These are hang overs from the source code I ported, editing these would require further TCL edits, which I'm trying to avoid (Never mind the refactor i did to get the variables out)